### DISCUSSION TITLE
Starship erases previous command's output if it's single line. Any workaround?

### DISCUSSION THREAD

**Discussion by vroad** [starship/starship#2892] [Q&A]
I installed starship to my MacBook Pro with nix, nix-darwin and home-manager's starship module.
I noticed that if previous command's output only contains single line, the command's output get rendered briefly and then overwritten by starship prompt.
I have been using starship with its default config.

for example: 
```
curl 'https://api.ipify.org?format=json'

~
❯
```

However, this works (as output is no longer single line thanks to jq):
```
curl -s 'https://api.ipify.org?format=json' | jq
{
  "ip": "x.x.x.x"
}

~
❯
``` 

starship --version output

```
starship --version
starship 0.54.0
branch:
commit_hash:
build_time:2021-07-10 20:29:37
build_env:rustc 1.52.1,
```

I've also tried 0.55.0 from nixpkgs-unstable (rather than nixpkgs-21.05-darwin), but the problem is not fixed yet.

```
starship 0.55.0
branch:
commit_hash:
build_time:2021-06-30 05:44:17
build_env:rustc 1.53.0,
```

**Comment by matchai** [comment]
Hi @vroad.
Could you please create a bug report using the `starship bug-report` command. This'll give us some additional insight into your system's configuration, giving us some added context to solve the problem. 😊

**Reply by vroad** [reply]
Thank you, I created an issue here.

https://github.com/starship/starship/issues/2901
